samples/nrf9160/modem_shell/prj.conf

```diff
@@ -128,6 +129,9 @@ CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS=17
 
 # Cellular positioning services
 CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+# Uncomment the following if MQTT is selected, for example when CONFIG_MOSH_CLOUD is set. Otherwise,
+# REST API is used for communication between the device and nRF Cloud.
+#CONFIG_NRF_CLOUD_CELL_POS=y
```

Comment:
Does MOSH compile if you have `CONFIG_NRF_CLOUD_MQTT` but not `CONFIG_NRF_CLOUD_CELL_POS`. Doesn't need to work but need to see if we should prevent that.

Reply:
The answer is no. I get this:
`/home/tor1/work/ncs/nrf/lib/multicell_location/services/nrf_cloud_integration.c:96: undefined reference to `nrf_cloud_cell_pos_request'`

I'll make BUILD_ASSERT to multicell_location library so the error becomes clear. I guess we don't have a way to handle this in multicell Kconfig.

Reply:
Dependencies are handled with:
https://github.com/nrfconnect/sdk-nrf/pull/6223
https://github.com/nrfconnect/sdk-nrf/pull/6244
Resolving this comment.